cmd/installer/cli/install_runpreflights.go

```diff
@@ -108,6 +109,10 @@ func runInstallPreflights(ctx context.Context, flags InstallCmdFlags, metricsRep
 		AssumeYes:            flags.assumeYes,
 		MetricsReporter:      metricsReported,
 	}); err != nil {
+		if errors.Is(err, os.ErrPermission) {
+			logrus.Errorf("Please make sure that the filesystem at %s is executable.", runtimeconfig.EmbeddedClusterHomeDirectory())
```

Comment:
this could be an inaccurate message though if it's the binary (or one of the subdirectories / parent directories) that isn't executable, right?

Reply:
Also, it might be executable for root, but not for group or other which is what's necessary for etcd for example.

Reply:
Ok, yeah I guess that's fine then. But I remember a discussion around etcd being run by a different user which might not have execute permissions on one of the folders in the path and I don't think this error will catch that, but I think that is a separate problem to this and can be fixed in a different PR.